src/pages/_app.tsx

```diff
@@ -7,14 +7,15 @@ import { LoadingSpinner } from "hds-react";
 import Error from "next/error";
 import { appWithTranslation } from "next-i18next";
 import { ToastContainer } from "react-toastify";
-import "nprogress/nprogress.css";
 
+import "nprogress/nprogress.css";
```

Comment:
Unexpected that this import has jumped into the in-app exports.

Reply:
Hmm, interesting that eslint didn't complain about that one, works on other imports